Update da-basic-plan-s1-infrastructure.md (#5208)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/remote/remote-access/directaccess/single-server-wizard/da-basic-plan-s1-infrastructure.md

    2.  If the corporate network is IPv6-based, the default address is the IPv6 address of DNS servers in the corporate network.

> [!NOTE]

> > Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy (NRPT).

Reviewer: Looks more proper to include all the words which the initialism is based on:
```suggestion
> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy Table (NRPT).
```
(if there is a rationale to only name "Name Resolution Policy", you may ignore this suggestion.)

> > If the DNS registration is needed, for e.g. Manage Out, it can be explicitely enabled with this registry key on the client:

Reviewer: Please note: use either A) "for example" or only "e.g." (exempli gratia, meaning "for example")
```suggestion
> If DNS registration is needed, for example by **Manage Out**, it can be explicitly enabled with this registry key on the client:
```
Also: typo correction, and add emphasis to the key phrase Manage Out.

>

> > Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`
> > Type: `DWORD`
> > Value name: `DisableNRPTForAdapterRegistration`
> > Values:
> > `1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)
> > `0` - DNS Registration enabled

Reviewer: Repeating what I noted in your PR #5207 : We need to add HTML <br/> to enable actual line breaks in a MarkDown Note blob (otherwise, all these lines will show up as one contiguous line).
```suggestion
> Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`<br/>
> Type: `DWORD`<br/>
> Value name: `DisableNRPTForAdapterRegistration`<br/>
> Values:<br/>
> `1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)<br/>
> `0` - DNS Registration enabled
```

> > Path: ```HKLM\System\CurrentControlSet\Services\Dnscache\Parameters```   
> > Type: ``DWORD``   
> > Value name: ``DisableNRPTForAdapterRegistration``   
> > Values:   
> > ``1`` - DNS Registration disabled (default since Windows 10 May 2020 Update)   
> > ``0`` - DNS Registration enabled

Reviewer: According to MarkDown best practice rules, only 1 single pair of code encapsulation backticks is needed for a horizontal (non-wrapped) string. 3 backticks can be used in a code block fence (multi-line). I have never seen a double pair used before, do you have a rationale for this uncommon usage? Let me suggest the common usage based on GitHub Flavored MarkDown (GFM):
```suggestion
> Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`   
> Type: `DWORD`
> Value name: `DisableNRPTForAdapterRegistration`
> Values:
> `1` - DNS Registration disabled (default since Windows 10 May 2020 Update)
> `0` - DNS Registration enabled
```
Also removed some redundant end-of-line whitespace (blanks).

> > [!NOTE]
> > Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy (NRPT).
> > If the DNS registration is needed, for e.g. Manage Out, it can be explicitely enabled with this registry key on the client:
> > 
> > Path: ```HKLM\System\CurrentControlSet\Services\Dnscache\Parameters```   
> > Type: ``DWORD``   
> > Value name: ``DisableNRPTForAdapterRegistration``   
> > Values:   
> > ``1`` - DNS Registration disabled (default since Windows 10 May 2020 Update)   
> > ``0`` - DNS Registration enabled

Reviewer: ```suggestion
> [!NOTE]
> Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy Table (NRPT).
> If DNS registration is needed, for example **Manage Out**, it can be explicitly enabled with this registry key on the client:
>
> Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`
> Type: `DWORD`
> Value name: `DisableNRPTForAdapterRegistration`
> Values:
> `1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)
> `0` - DNS Registration enabled
```
